debezium-core/src/main/java/io/debezium/relational/history/TableChanges.java

```diff
@@ -36,11 +36,16 @@ public TableChanges() {
         this.changes = new ArrayList<>();
     }
 
-    public static TableChanges fromArray(Array array) {
+    /*
+     * useCatalogBeforeSchema is true if the parsed string contains only 2 items and the first should be used as
```

Comment:
Could you make this an actual method comment (i.e. leading with `/**` and also `@param useCatalogBeforeSchema`).

Reply:
I'll add the missing `@param`.